### PR TITLE
feat: Added functionality to delete an activity

### PR DESCRIPTION
**Pull request by sechaba-mache**

### CODE REVIEW

File: angular-holiday-planner/src/app/components/trip-tile/trip-tile.component.html
Comment (CharelMarais) [inline angular-holiday-planner/src/app/components/trip-tile/trip-tile.component.html:1]: You are using Arbitrary values that are being reused, I would suggest that you create custom values in your tailwind config file for any arbitrary values that are used more than once.

**Review by CharelMarais — APPROVED** [review]
Please review my comment, the issue is not major so I still approved the request.

File: angular-holiday-planner/src/app/components/trip-tile/trip-tile.component.html
Comment (kphutiyagae) [reply angular-holiday-planner/src/app/components/trip-tile/trip-tile.component.html:1]: Yes, what Charel said.

**Review by kphutiyagae — APPROVED** [review]

File: angular-holiday-planner/src/app/components/trip-tile/trip-tile.component.html
Comment (sechaba-mache) [reply angular-holiday-planner/src/app/components/trip-tile/trip-tile.component.html:1]: Understood. This component is probabaly going to be scrapped but if not then I'll look into your suggestions. I appreciate the comments

**Pull request merged**